testing/test_aux.out

```diff
@@ -12,10 +12,16 @@ RESULT pass protocol.bacnet.version
 RESULT fail network.brute
 RESULT pass network.brute
 RESULT skip network.brute
+RESULT fail security.tls.v3
```

Comment:
so there should be three different result codes here, there's pass (good cert), fail (bad cert), and then none (?, no 443 open). The three faux devices are configured differently so they should have different results.  That's how the integration tests actually validate that your code is doing the right thing.

Reply:
All good I believe I have implemented skip to take care of the no 443 open case.